index.js

```diff
@@ -43,7 +43,16 @@ var excludedKeys = {
 	$webkitStorageInfo: true,
 	$window: true
 };
+var isChromeExtension = (function () {
+	/* global chrome */
+	return (typeof chrome !== 'undefined') &&
+		(typeof chrome.runtime !== 'undefined') &&
+		(typeof chrome.runtime.id !== 'undefined');
+}());
 var hasAutomationEqualityBug = (function () {
+	if (isChromeExtension) {
+		return false;
```

Comment:
Nope. It's shown even with console.error rewritten.

It seems that it runs "inside" `extensions::platformApp`, which has a separate context.

To tell you the exact line - it seems to be thrown here, line 17

https://chromium.googlesource.com/chromium/src/+/master/extensions/renderer/resources/platform_app.js#17

They seem to have some notes here

https://chromium.googlesource.com/chromium/src/+/master/extensions/renderer/resources/platform_app.js#43